scalableminds/webknossos
# Add TS types for search inputs

Small PR. I added and fixed some TS type for event callbacks of the `<Search>` input. We are using a live, search-on-every-change approach so every `onPressEnter` event handler was unused anyway. However, binding `onPressEnter` and `onChange` events to the same handler function messes with typing. So I removed the unused `onPressEnter` events.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Run type check
- Go to one of the admin pages and make sure that search still works.


------
(Please delete unneeded items, merge only when none are left open)
